app/Http/Controllers/searchController.php

```diff
@@ -16,6 +16,23 @@ public function search($searchKey)
     {
 		$products = Product::search($searchKey)->get();
 		$searchWord = '$searchKey';
-		return view('search',['searchWord'=>$searchKey], compact('products'));
+		return view(
+			'search',
+			[
+				'searchWord'=>$searchKey
+			],
+			compact('products'));
```

Comment:
what does compact('products') do?

Reply:
It creates an array of variables and their values. In this case, value stored in variable 'product's which is created in line 17 will be returned (displayed)

Reply:
it creates an array of the results 'products'